test/complex-json.spec.js

```diff
@@ -0,0 +1,111 @@
+/**
```

Comment:
This complex input was covered in an integration test, was simple to make a unit test using it as well.